src/Documentation/UserManual/DeviceGenericSerial.dox

```diff
@@ -6,7 +6,8 @@ It does not collect tracking or image data, only allows sending and receiving da
 using OpenIGTLink commands (see \ref PlusServerCommands).
 
 A (customizable) newline string is appended after each sent command and the device listens
-for reponse that is terminated by the same newline string.
+for reponse that is terminated by the same newline string by default.
+The default can be modified by specifying "acceptReply" parameter of ReplyTermination type.

```

Comment:
Ok, is there also going to be a device class for those motors?

Now that I understand, a method argument is fine. Perhaps a small comment as to which device class needs the behavior, and that different replies have different format for that device.